pontoon/sync/changeset.py

```diff
@@ -147,6 +147,12 @@ def update_entity_translations_from_vcs(
             )
         approved_translations = []
 
+        # Unfuzzy existing fuzzy translations to prevent duplicate fuzzies.
+        for translation in db_translations:
```

Comment:
Is that an optimization issue or could it cause more serious problems?

In `old_translations` we only keep approved ones.